Symfony/CS/Tests/ApplicationTest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Tests;
+
+use Symfony\CS\Console\Application;
+
+/**
+ * @author SpacePossum
```

Comment:
[it is valid](https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc.md#examples-1)